connectors/google/google-gemini/src/main/java/io/camunda/connector/gemini/model/GeminiRequestData.java

```diff
@@ -0,0 +1,350 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.gemini.model;
+
+import io.camunda.connector.feel.annotation.FEEL;
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+public record GeminiRequestData(
+    @TemplateProperty(
+            label = "Project ID",
+            group = "input",
+            description = "Project identifier.",
+            feel = Property.FeelMode.disabled)
+        @NotNull
+        String projectId,
+    @TemplateProperty(
+            label = "Region",
+            group = "input",
+            description = "Input region.",
+            feel = Property.FeelMode.disabled)
+        @NotNull
+        String region,
+    @TemplateProperty(
+            label = "Model",
+            group = "input",
+            description = "Select gemini model.",
+            feel = Property.FeelMode.disabled,
+            choices = {
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_5_FLASH_001",
+                  label = "gemini-1.5-flash-001"),
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_5_FLASH_002",
+                  label = "gemini-1.5-flash-002"),
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_5_PRO_001",
+                  label = "gemini-1.5-pro-001"),
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_5_PRO_002",
+                  label = "gemini-1.5-pro-002"),
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_0_PRO_001",
+                  label = "gemini-1.0-pro-001"),
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_0_PRO_002",
+                  label = "gemini-1.0-pro-002"),
+              @TemplateProperty.DropdownPropertyChoice(
+                  value = "GEMINI_1_0_PRO_VISION_001",
+                  label = "gemini-1.0-pro-vision-001")
+            })
+        @NotNull
+        ModelVersion model,
+    @FEEL
+        @TemplateProperty(
+            label = "Prompt",
+            group = "input",
+            description = "Insert prompt.",
+            feel = Property.FeelMode.required)
+        @NotNull
+        List<Object> prompts,
```

Comment:
I'm never comfortable being able to log the prompts. Do we have a decision on that? It affects all the AI connectors